source/rmarkdown/analyses_diversity/_child_model_observed_richness.Rmd

```diff
@@ -628,18 +794,334 @@ for (i in efvars) {
 p
 ```
 
-global c_density effect (IRR per IQR), with 95% CI. This gives a global (averaged) slope across the different land-use categories
+global log_c_density effect. This gives a global (averaged) slope across the different land-use categories
+
+```{r log-c-density-lowest_aic-{{taxon_group}}-{{primerset}}-{{unit}}}
+
+# Slope for log(c_density) on the latent (log) scale
+tr_add <- emtrends(lowest_aic, ~ 1, var = "log_c_density", mode = "latent")
+tr_add_sum <- summary(tr_add, infer = TRUE)
+
+z <- qnorm(0.975)
+
+coef_tbl <- as_tibble(tr_add_sum) %>%
+  transmute(beta = log_c_density.trend, SE = SE, p.value = p.value)
+
+inc_tbl <- tibble(
+  increment = c("×e (e-fold)", "×2 (doubling)", "×10 (tenfold)", "×1.2 (+20%)"),  
+  delta     = c(1,               log(2),          log(10),         log(1.2)),
+  code      = c("e",             "2x",            "10x",           "1.2x")   # NEW      
+)
+
+result <- crossing(coef_tbl, inc_tbl) %>%
+  mutate(
+    IRR = exp(beta * delta),
```

Comment:
Seems correct. Should be the same as `k^beta` where k is the k-fold change, i.e. `k = exp(delta)` in your code.